docs/src/domains/in-depth/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
As "in-depth" is not in the name anymore, it shouldn't be in the in-depth path.  It can go back to steps.

Reply:
Moved